source/modulo_components/component_descriptions/modulo_component.json

```
@@ -8,12 +8,20 @@
   "registration":  "modulo_components::Component",
   "inherits": "",
   "parameters": [
+    {
+      "display_name": "Rate",
+      "description": "The frequency in Hertz for all periodic callbacks (this parameter overrides the period parameter)",
```

actually one last nitpick: since the rate parameter is now the "public" one and period is "internal", it's a bit weird to have this in the public description.
```suggestion
      "description": "The frequency in Hertz for all periodic callbacks",
```